src/code/z_parameter.c

```diff
@@ -7,6 +7,13 @@
 #include "overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope.h"
 #include "overlays/actors/ovl_En_Mm3/z_en_mm3.h"
 
+typedef enum {
+    /* 0 */ PICTO_STATE_OFF,         // Not using the pictograph
+    /* 1 */ PICTO_STATE_LENS,        // Looking through the lens of the pictograph
+    /* 2 */ PICTO_STATE_SETUP_PHOTO, // Looking at the photo currently taken
+    /* 3 */ PICTO_STATE_PHOTO
+} PictoState;
```

Comment:
```suggestion
    /* 0 */ PICTO_BOX_STATE_OFF,         // Not using the pictograph
    /* 1 */ PICTO_BOX_STATE_LENS,        // Looking through the lens of the pictograph
    /* 2 */ PICTO_BOX_STATE_SETUP_PHOTO, // Looking at the photo currently taken
    /* 3 */ PICTO_BOX_STATE_PHOTO
} PictoBoxState;
```
This refers to the item right?